indico/modules/rb/controllers/backend/admin.py

```diff
@@ -532,9 +532,9 @@ def _process_POST(self, args):
 
 
 class RHMapAreas(RHRoomBookingAdminBase):
-    @use_args(_create_args)
-    def _process_POST(self, args):
-        create_area(args)
+    @use_kwargs(_create_args)
```

Comment:
You can just do `create_area(**args)` below without changing anything here - no need to convert between dicts and function args before calling that function ;)

Reply:
Oh yeah, I think I forgot to change it after I experimented with `use_kwargs`. Thanks for fixing it!